about.html

```diff
@@ -4,9 +4,25 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="./main.css">
     <title>Document</title>
 </head>
-<body>
-    
+<body class="aboutBody">
+    <div class="img-fluid">
+        <img class="spanImg" src="./bandpic.png" alt="picture of the band">
+        <h1 class="aboutHeading">About Scuurvy and Band History</h1>
```

Comment:
add this for the h1 so they are all uniform   <h1 class="d-flex justify-content-center">About Scuurvy</h1>

Reply:
`<h1 class="d-flex justify-content-center">About</h1>`

Reply:
What do you mean add "this"

Reply:
want me to create a h1 css for all other pages?